### ISSUE TITLE
Remove `chain-spec-generator`

### ISSUE THREAD

**Issue by bkontur** [polkadot-fellows/runtimes#395]
Where possible, we should use `chain-spec-builder` from Substrate, which can be downloaded from the `polkadot-sdk` [release page assets](https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.14.0) or from [crates.io](https://crates.io/crates/staging-chain-spec-builder).

Relates to: https://github.com/paritytech/polkadot-sdk/pull/4518

## TODO
- [ ] fix bridges zombienet tests - https://github.com/polkadot-fellows/runtimes/blob/main/integration-tests/bridges/environments/polkadot-kusama/bridge_hub_kusama_local_network.toml#L8
- [ ] zombienet-sdk-tests - https://github.com/polkadot-fellows/runtimes/blob/main/integration-tests/zombienet/src/lib.rs#L10
- [ ] anything else?

**Comment by kianenigma** [comment]
Seems like a "clear" task that can be mentored. Should we have a mentor tag here as well?

**Comment by bkontur** [comment]
> Seems like a "clear" task that can be mentored. Should we have a mentor tag here as well?

yes, sure, but I cannot update tags/labels here. Do we even use tags/labels for the fellows repo?

**Comment by kianenigma** [comment]
@anaelleltd what would you suggest? would be good to give label altering rights to existing fellowship members. 

Perhaps you can create github teams from different rank. Based on this, then you can give more proper merge/label and such permissions to members.

**Comment by anaelleltd** [comment]
> @anaelleltd what would you suggest? would be good to give label altering rights to existing fellowship members.

Yes, Bryan also mentioned something along these lines for the RFCs repo; so as to update the status of each RFC.

> Perhaps you can create github teams from different rank. Based on this, then you can give more proper merge/label and such permissions to members.

I only have "Member" status in the Fellowship repo, so I will have to speak to Raul or Rob (the current repo owners) about this new setup. I will also ask them to give me team maintainer right for future admin duties.

From what I read, teams would be useful for ping-ing members of higher ranks to do review, at the very least. 
When it comes to giving members **[triage/write](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role)** rights on the Fellowship repo, this will probably have to be done on a case-by-case basis. I wouldn't recommend adding all Fellowship members to teams at once. 

Would this work from your end or do you have any other requirements in mind?

**Comment by Doordashcon** [comment]
@kianenigma @bkontur @anaelleltd If no one else is already working on this I would like to pick this up!
